### PR TITLE
Feat(eos_designs): Add support for variable inputs for source_vrf in ip_helper

### LINKED ISSUE #6745: Support for the “use_default_mgmt_method_vrf” knob with DHCP relay
### Enhancement summary

Enhance current DHCP relay configuration in network services in eos_designs to support the knob "use_default_mgmt_method_vrf":
```
ip_helpers:
            # IPv4 DHCP server IP.
          - ip_helper: <str; required; unique>
           # Interface name.
           source_interface: <str>
           # VRF to originate DHCP relay packets to DHCP server. If not set, uses current VRF.
            source_vrf: use_default_mgmt_method_vrf <<-----
```

This will automatically use the OOB management VRF or the in-band management VRF, where the DHCP server is typically reachable.

### Which component of AVD is impacted

eos_designs

### Use case example

Enhance DHCP relay configurations to use the knob "use_default_mgmt_method_vrf". This allows to set the same configuration whether the fabric is OOB, Inband, etc since eos_designs will automatically configure it accordingly.

### Describe the solution you would like

See above

### Describe alternatives you have considered

currently configuring explicitly the VRF name

### Additional context

_No response_

### Contributing Guide

- [x] I agree to follow this project Code of Conduct

### PR DESCRIPTION
**Pull request by laxmikantchintakindi**
## Change Summary

Add support for variable inputs for source_vrf and source_interface in ip_helper.
source_vrf can be `use_mgmt_interface_vrf` or `use_inband_mgmt_vrf` or `use_default_mgmt_method_vrf` or any other string or None.
The value of the source vrf is calculated as per the input.
source_interface can be `use_mgmt_interface` or `use_inband_mgmt_interface` or `use_default_mgmt_method_interface` or any other string or None.

e.g. 
```
ip_helpers:
  - ip_helper: 1.1.1.1
    source_interface: use_inband_mgmt_interface
    source_vrf: use_inband_mgmt_vrf
```

## Related Issue(s)

Fixes #6745

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Enhance DHCP relay configurations to use the knob "use_default_mgmt_method_vrf". This allows to set the same configuration whether the fabric is OOB, Inband, etc since eos_designs will automatically configure it accordingly.

## How to test
Add use_default_mgmt_method_vrf/use_inband_mgmt_vrf for source_vrf in ip_helpers and check the generated VRF.
### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.arista.com/stable/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/extensions/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py [hunk 1]
```diff
@@ -509,10 +509,15 @@ def get_additional_svi_config(
         ip_helpers = svi.ip_helpers or vrf.ip_helpers
         if ip_helpers:
             for svi_ip_helper in ip_helpers:
+                source_vrf = svi_ip_helper.source_vrf
+                source_interface = self.get_source_interface_for_ip_helpers(svi_ip_helper.source_interface)
+                source_vrf = (
+                    self.get_vrf(svi_ip_helper.source_vrf, context=f"{vrf.name}.source_vrf or {vrf.name}.svis[{svi.name}].source_vrf") if source_vrf else None
+                )
```
Comment (laxmikantchintakindi) [inline python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py]:
```suggestion
                source_interface = self.get_source_interface_for_ip_helpers(svi_ip_helper.source_interface)
                source_vrf = (
                    self.get_vrf(svi_ip_helper.source_vrf, context=f"{vrf.name}.source_vrf or {vrf.name}.svis[{svi.name}].source_vrf") if svi_ip_helper.source_vrf else None
                )
```

**Review by Shivani-gslab — APPROVED** [review]
LGTM

File: python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py [hunk 2]
```diff
@@ -509,10 +509,16 @@ def get_additional_svi_config(
         ip_helpers = svi.ip_helpers or vrf.ip_helpers
         if ip_helpers:
             for svi_ip_helper in ip_helpers:
+                source_interface = self.get_source_interface_for_ip_helpers(svi_ip_helper.source_interface)
+                source_vrf = (
+                    self.get_vrf(svi_ip_helper.source_vrf, context=f"{vrf.name}.source_vrf or {vrf.name}.svis[{svi.name}].source_vrf")
```
Comment (MaheshGSLAB) [inline python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py]: What if user set the source_vrf at both vrf and svi level. As per line 509 svi should take the precedence but in context we are giving the preference to vrf. Lets change it and give the preference to svi instead of vrf

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+l3spine:
+  defaults:
+    bgp_as: 65001
+    spanning_tree_mode: none
+    loopback_ipv4_pool: 192.0.255.0/24
+    uplink_ipv4_pool: 10.255.1.0/24
+    mlag_interfaces: [Ethernet10]
+    mlag_peer_ipv4_pool: "10.10.10.0/24"
+    mlag_peer_l3_ipv4_pool: "10.10.20.0/24"
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml]: Do we really need mlag configuration to test the ip_helpers?

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_D.yml [hunk 4]
```diff
@@ -114,6 +114,10 @@ tenant_d:
             tags: ['v6wan']
             enabled: true
             profile: GENERIC_DUAL_STACK
+        ip_helpers:
+          - ip_helper: 1.1.1.3
+            source_interface: use_default_mgmt_method_interface
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_D.yml:119]: Do we really need this test, as use_default_mgmt_method_interface already tested in your new group var file

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        svis:
+          - id: 11
+            name: SVI2
+            ip_helpers:
+              - ip_helper: 1.1.1.4
+                source_interface: use_inband_mgmt_interface
+                source_vrf: use_inband_mgmt_vrf
+              - ip_helper: 1.1.1.5
+                source_interface: vlan100
+                source_vrf: use_inband_mgmt_vrf
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml:83]: lets give a specific vrf aprt from use_inband_mgmt_vrf, use_mgmt_interface_vrf or use_default_mgmt_method_vrf

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/intended/configs/network-services-ip-helpers-3.cfg [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+interface Loopback0
+   description ROUTER_ID
+   no shutdown
+   ip address 192.0.255.11/32
+!
+interface Vlan11
+   description SVI2
+   shutdown
+   vrf IP_HELPERS3
+   ip helper-address 1.1.1.4 vrf INBANDMGMT source-interface vlan100
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/intended/configs/network-services-ip-helpers-3.cfg]: we says source-interface as vlan 100 but there is no vlan100 interface created?

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml (hunk 5 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml:83]: We already have this in DC1_TENANT_NETWORKS.

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml (hunk 3 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml]: Removed.

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_D.yml (hunk 4 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_D.yml:119]: This is at vrf level.

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/intended/configs/network-services-ip-helpers-3.cfg (hunk 6 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/intended/configs/network-services-ip-helpers-3.cfg]: Fixed inband mgmt confiurations.

File: python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py (hunk 2 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py]: Updated and improved the context.

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_D.yml (hunk 4 above)
Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_D.yml:119]: why not add one more host in your IP_HELPERS group vars

File: python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py [hunk 7]
```diff
@@ -508,11 +508,21 @@ def get_additional_svi_config(
...
         ip_helpers = svi.ip_helpers or vrf.ip_helpers
         if ip_helpers:
-            for svi_ip_helper in ip_helpers:
+            for idx, svi_ip_helper in enumerate(ip_helpers):
+                # Not enforcing default_mgmt_method_vrf or interface when it is not defined in inputs.
+                source_interface = self.get_local_interface(svi_ip_helper.source_interface) if svi_ip_helper.source_interface else None
+                source_vrf = (
+                    self.get_vrf(
+                        svi_ip_helper.source_vrf,
+                        context=f"{vrf.name}.svis[{svi.name}].ip_helpers[{idx}].source_vrf or {vrf.name}.ip_helpers[{idx}].source_vrf or",
```
Comment (MaheshGSLAB) [inline python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py]: lets add negative test on this context and i think last `or` is by mistake.

Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py]: Added.

File: python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py [hunk 8]
```diff
@@ -508,11 +508,24 @@ def get_additional_svi_config(
...
-            for svi_ip_helper in ip_helpers:
+            for idx, svi_ip_helper in enumerate(ip_helpers):
+                # Not enforcing default_mgmt_method_vrf or interface when it is not defined in inputs.
+                source_interface = self.get_local_interface(svi_ip_helper.source_interface) if svi_ip_helper.source_interface else None
+                source_vrf = (
+                    self.get_vrf(
+                        svi_ip_helper.source_vrf,
+                        context=(
+                            f"tenants[name={tenant.name}].vrfs[name={vrf.name}].svis[{svi.name}].ip_helpers[{idx}].source_vrf or"
+                            f" tenants[name={tenant.name}].vrfs[name={vrf.name}].ip_helpers[{idx}].source_vrf"
```
Comment (MaheshGSLAB) [inline python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py]:
```suggestion
                            f"tenants[name={tenant.name}].vrfs[name={vrf.name}].svis[name={svi.name}].ip_helpers[ip_helper={idx}].source_vrf or"
                            f" tenants[name={tenant.name}].vrfs[name={vrf.name}].ip_helpers[ip_helper={idx}].source_vrf"
```

Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py]: removed idx and used ip_helper name.

**Review by MaheshGSLAB — APPROVED** [review]

**Review by alexeygorbunov — COMMENTED** [review]
/

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml [hunk 9]
```diff
@@ -533,10 +533,26 @@ $defs:
...
                       type: str
-                      description: Interface name.
+                      description: |-
+                        Interface name to originate DHCP relay packets to DHCP server.
+                        The value will be interpreted according to these rules:
+                        - `use_mgmt_interface` will configure the OOB management interface as the source interface.
+                        - `use_inband_mgmt_interface` will configure the `inband_mgmt_interface` as the source interface.
+                        - `use_default_mgmt_method_interface` will configure the source interface for one of the two options above depending on the value of `default_mgmt_method`.
+                        - Any other string will be used directly as the source interface.
+                        - If not set, EOS uses the VRF on the SVI.
```
Comment (alexeygorbunov) [inline python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml]: I'm not sure this is a valid statement. If not set - EOS will set giaddr based on the "receiving" interface (like SVI) but outbound interface will be picked based on the routing table of the VRF (of the "source_vrf" if set, otherwise of the VRF of the receiving interface).
Maybe we should just drop this line?

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_svi_settings.schema.yml [hunk 10]
```diff
@@ -120,10 +120,26 @@ $defs:
...
               type: str
-              description: Interface name to originate DHCP relay packets to DHCP server.
+              description: |-
+                Interface name to originate DHCP relay packets to DHCP server.
+                The value will be interpreted according to these rules:
+                - `use_mgmt_interface` will configure the OOB management interface as the source interface.
+                - `use_inband_mgmt_interface` will configure the `inband_mgmt_interface` as the source interface.
+                - `use_default_mgmt_method_interface` will configure the source interface for one of the two options above depending on the value of `default_mgmt_method`.
+                - Any other string will be used directly as the source interface.
+                - If not set, EOS uses the VRF on the SVI.
```
Comment (alexeygorbunov) [inline python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_svi_settings.schema.yml]: Same concerns here

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/hosts.yml [hunk 11]
```diff
@@ -450,6 +450,7 @@ all:
             net-srv-l3-po-sub-interface-without-ip:
             net-srv-l3-po-sub-interface-without-main-po:
             net-srv-l3-po-without-member-interfaces:
+            network-services-ip-helpers-leaf1:
```
Comment (alexeygorbunov) [inline ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/hosts.yml]: Should hostname reflect the cause of the error/exception?

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml [hunk 12]
```diff
@@ -0,0 +1,2 @@
+---
+default_mgmt_method: inband
```
Comment (alexeygorbunov) [inline ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: Is there any tested use case which relies on this setting right now?

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: This is required to generate inband management interface when source_interface is set to use_inband_mgmt_interface.

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/hosts.yml (hunk 11 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/hosts.yml]: Renamed to missing-inband-mgmt-config-ip-helpers.

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_svi_settings.schema.yml (hunk 10 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_svi_settings.schema.yml]: Removed the line.

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml (hunk 9 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml]: Removed the line.

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml (hunk 12 above)
Comment (alexeygorbunov) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: Hm, but `inband_mgmt_interface` is explicitly set for `network-services-ip-helpers-leaf3`.

```
    - name: network-services-ip-helpers-leaf3
      id: 11
      inband_mgmt_vrf: INBANDMGMT
      inband_mgmt_interface: vlan100 <<<<<<<<
      filter:
        tenants: ["TEST2"]
        always_include_vrfs_in_tenants: ["TEST2"]
```

Deleting `ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml` leads to no changes in eos_designs_unit_tests.

Comment (gmuloc) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: @laxmikantchintakindi can you please take a look atthis?

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: In network-services-ip-helpers-leaf3, vrf and source interfaces are configured on the ip_helpers using `use_inband_mgmt_interface` and `use_inband_mgmt_vrf` as inputs in tenant-  TEST2.

Comment (gmuloc) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: so this file is not useful and can be deleted it seems (that was Alexey's point)

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: Removed the host, but this PR doesn't test the ip_helpers with `source_vrf`: use_inband_mgmt_vrf, `source_interface: use_inband_mgmt_interface`.

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: We are not losing the coverage here but this inputs are not covered. leaf3 was creating below config:
```
interface Vlan11
   description SVI2
   shutdown
   vrf IP_HELPERS3
   ip helper-address 1.1.1.4 vrf INBANDMGMT source-interface vlan100 
   ip helper-address 1.1.1.5 vrf INBANDMGMT source-interface vlan100
```
Here INBANDMGMT and vlan100 are generated using get_vrf_and_source_interface logic.

Comment (gmuloc) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: I think the point Alexey was making was that with or without the hostvars file, there was no change about the output so he was questioning why tou are keeping the hostvar file at all. He was not asking to remove the device, only the hostvars file

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-ip-helpers-leaf3.yml]: Sorry for the confusion! Reverted the change and removed the host_vars for leaf3.

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_svi_settings.schema.yml [hunk 13]
```diff
@@ -134,10 +134,25 @@ $defs:
...
+              description: |-
+                VRF to originate DHCP relay packets to DHCP server.
+                The value will be interpreted according to these rules:
+                - `use_mgmt_interface_vrf` will configure the `mgmt_interface_vrf` as the source VRF.
+                  An error will be raised if `mgmt_ip` or `ipv6_mgmt_ip` are not configured for the device.
+                - `use_inband_mgmt_vrf` will configure the `inband_mgmt_vrf` as the source VRF.
+                  An error will be raised if inband management is not configured for the device.
+                - `use_default_mgmt_method_vrf` will configure the source VRF for one of the two options above depending on the value of `default_mgmt_method`.
+                - Any other string will be used directly as the source VRF name.
+                - If not set, EOS uses the VRF on the SVI.
```
Comment (alexeygorbunov) [inline python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_svi_settings.schema.yml:155]: Similar key under network_services' VRF definition is missing this line. Should we add it there to make sure description of ip_helpers is in sync for vrfs and vrfs[].svis[]?

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/host_vars/flow-tracking-missing-tracker.yml [hunk 14]
```diff
@@ -5,6 +5,13 @@ l2leaf:
       id: 1
       # Could as well be hardware
       flow_tracker_type: sampled
+      uplink_interfaces: [Ethernet2, Ethernet3]
+      uplink_switch_interfaces: [Ethernet1, Ethernet2]
+      inband_mgmt_vlan: 100
+      inband_mgmt_subnet: 192.168.101.0/24
+      inband_mgmt_vrf: INBANDMGMT
+      # Set to create inband mgmt config for missing-inband-mgmt-config-ip-helpers.
+      uplink_switches: [missing-inband-mgmt-config-ip-helpers, missing-inband-mgmt-config-ip-helpers]
```
Comment (alexeygorbunov) [inline ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/host_vars/flow-tracking-missing-tracker.yml]: Are these changes needed for `missing-inband-mgmt-config-ip-helpers`'s test? Commenting out these lines seem to not change any results of the negative test.
P.S. If we indeed need a parent device then maybe we should use/create new device to not mix negative unit tests (flow tracking with ip-helpers to not have them dependant)?

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+      mgmt_ip: 10.1.1.1/24
+      filter:
+        tenants: ["TEST1"]
+        always_include_vrfs_in_tenants: ["TEST1"]
+    - name: network-services-ip-helpers-leaf2
+      id: 2
+      mgmt_ip: 10.1.1.2/24
+      filter:
+        tenants: ["TEST1"]
+        always_include_vrfs_in_tenants: ["TEST1"]
```
Comment (alexeygorbunov) [inline ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml]: Why do we need both network-services-ip-helpers-leaf1 and network-services-ip-helpers-leaf2? Can't see any difference between use cases that they test

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_svi_settings.schema.yml (hunk 13 above)
Comment (laxmikantchintakindi) [reply python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_svi_settings.schema.yml:155]: Added.

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/host_vars/flow-tracking-missing-tracker.yml (hunk 14 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/host_vars/flow-tracking-missing-tracker.yml]: Removed new keys. Yes this change was not needed.

File: ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml (hunk 15 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/extensions/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_SERVICES_IP_HELPERS.yml]: Remove 2nd host.

**Review by alexeygorbunov — APPROVED** [review]
LGTM
